src/lib.rs

```diff
@@ -342,8 +342,8 @@ mod alloc {
     #[cfg(feature = "std")]
     pub(crate) use std::boxed::Box;
 
-    #[cfg(not(feature = "std"))]
-    pub(crate) use alloc::string::String;
+    // #[cfg(not(feature = "std"))]
```

Comment:
I think just remove this rather than commenting it out, and the one below if you can.